js/shared.js

```diff
@@ -231,7 +149,7 @@ export const sendEmailLink = () => {
         },
         body: JSON.stringify({ email: signInEmail, continueUrl }),
     }).then(() => {
-        signInFlowRender(signInEmail)
+        signInFlowRender(signInEmail);
```

Comment:
Yes and No.
Having an error handler is better. But I suggest it be done at a later time, because of below reasons:
- It's not sufficient to simply add an error handler that prints error message. When an error occurs, a different UI needs to be displayed, and we haven't decided the UI yet.
- The code here is part of implementation done by another member, and it's better to let the same person do updates.

Reply:
I'll approve, then.